cli.py

```
@@ -1,15 +1,45 @@
+"""
+Interface layer: CLI, REPL, and event adapters.
+"""
+
 import typer
 from core import repl, summarizer
+from codechat.events.event_bus import bus
```

Ruff (0.12.2)

`codechat.events.event_bus.bus` imported but unused

Remove unused import: `codechat.events.event_bus.bus`

(F401)